src/tutorial.md

```diff
@@ -0,0 +1,1056 @@
+      >
+        Withdraw to BTC
+      </button>
+    </form>
+  );
+}
+```
+
+If you run `npm run dev` from the `frontend` folder you should see the UI display.
+
+What we have here is a basic UI for converting BTC and sBTC, and borrowing and lending sBTC.
+
+Now that we have our basic UI in place, let's add functionality one piece at a time.
+
+## Initiating a sBTC Deposit
+
+The first thing we are going to do is create a component to initiate a sBTC deposit.
+
+You should already be familiar with how sBTC works at a high level, but what we are going to be doing is constructing a custom Bitcoin transaction that will have all the data we need in order to successfully deposit it into the threshold signature wallet and then mint our sBTC.
+
+Remember that for the Developer Release, the system that actually does the minting is not the fully decentralized version, it is a centralized single binary, but for the purposes of interacting with it as an application developer, the interface will be very similar to the final version.
+
+Recall that in order to mint sBTC to our Stacks address we need to deposit the amount of BTC we want to convert into the threshold signature wallet, and pass in what Stacks address we want the sBTC minted to in via the OP_RETURN data.
+
+The protocol and sBTC Clarity contracts will handle the actual minting of the sBTC.
+
+We can use the sBTC package to make constructing that transaction much easier, and then we can use Leather's API to broadcast it.
+
+We'll start by installing the sBTC package.
+
+```bash
+npm install sbtc
+```
+
+Let's now update the `DepositForm.js` component.
+
+```jsx
+"use client";
+
+import { useState, useContext } from "react";
+import {
+  DevEnvHelper,
+  sbtcDepositHelper,
+  TESTNET,
+  TestnetHelper,
+  WALLET_00,
+  WALLET_01,
+} from "sbtc";
+import { bytesToHex, hexToBytes } from "@noble/hashes/utils";
+import * as btc from "@scure/btc-signer";
+
+import { UserContext } from "../UserContext";
+
+export default function DepositForm() {
+  const { userData } = useContext(UserContext);
+  const [satoshis, setSatoshis] = useState("");
+
+  const handleInputChange = (event) => {
+    setSatoshis(event.target.value);
+  };
+
+  const buildTransaction = async (e) => {
+    e.preventDefault();
+    const testnet = new TestnetHelper();
+    // const testnet = new DevEnvHelper();
+
+    // setting BTC address for devnet
+    // const bitcoinAccountA = await testnet.getBitcoinAccount(WALLET_00);
+    // const btcAddress = bitcoinAccountA.wpkh.address;
+    // const btcPublicKey = bitcoinAccountA.publicKey.buffer.toString();
+
+    // setting BTC address for testnet
+    const btcAddress = userData.profile.btcAddress.p2wpkh.testnet;
+    const btcPublicKey = userData.profile.btcPublicKey.p2wpkh;
+
+    let utxos = await testnet.fetchUtxos(btcAddress);
+
+    // If we are working via testnet
+    // get sBTC deposit address from bridge API
+    const response = await fetch(
```

Comment:
Remove dependency on sbtc.tech here. The sBTC wallet address should be configurable or a constant for sBTC DR 0.1 
Alternatively, we can read it from the contract directly.

Reply:
@friedger What is the deployed address of the developer release contract on testnet?

Reply:
https://github.com/stacks-network/sbtc/issues/262#issuecomment-1760140843